core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -82,7 +82,8 @@ object ConsoleConsumer extends Logging {
       reportRecordCount()
 
       // if we generated a random group id (as none specified explicitly) then avoid polluting zookeeper with persistent group data, this is a hack
-      if (!conf.groupIdPassed)
+      // NOTE : this is true only for the Old consumer, because the New one doesn't create an entry to "/consumers"
```

Comment:
I don't think you need this additional comment. It's clear that only the old consumer uses ZK.

Reply:
Yes, maybe it's useless.